pyproject.toml

```diff
@@ -39,7 +39,8 @@ dask = { version = "^2021.2.0", optional = true, extras = [
   "array",
   "dataframe"
 ] }
-geoalchemy2 = { version = ">=0.6,<0.11", optional = true }
+datafusion = { version = ">=0.4", optional = true }
```

Comment:
You need to add `maturin` as a development dependency so that the nix development environment works. This is precisely why the nix setup exists, to catch underspecified dependencies :)

You can do this with `poetry add maturin --dev`

Reply:
Updated.